website/docs/cdktf/concepts/constructs.mdx

```diff
@@ -51,7 +51,7 @@ You can import any [CDKTF-compatible](#available-constructs) construct that is a
 The following TypeScript example instantiates a construct called `KubernetesWebAppDeployment` and uses the available arguments to specify that the deployment will have two replicas.
```

Comment:
I'm so excited to see this! One thing I want to say is, we probably want to remove "TypeScript" from the text above all the examples where we have more than one language. so it should say "The following example" instead :)

Nice work!

Reply:
Good catch! Updated 😁